### PR TITLE
feat: screen mixin that sets status bar color

### PR DESCRIPTION
**Pull request by BeeMargarida**
| - | - |
| --- | --- |
| Issue | https://github.com/ripe-tech/ripe-id-mobile/pull/47#discussion_r680867702 |
| Dependencies | -- |
| Decisions | Mixin that handles the status bar color change. |
| Animated GIF | -- |

### CODE REVIEW

**Comment by gcandal** [comment]
https://github.com/ripe-tech/ripe-id-mobile/pull/47/files#r681138111

File: react/util/mixins/screen.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+ * @returns {Class} The new class with the added screen functionality.
+ */
+export const ScreenMixin = superclass =>
+    class extends superclass {
+        /**
+         * Sets the status bar color in Android platforms.
+         */
+        setStatusBar() {
+            if (Platform.OS === "android") {
+                StatusBar.setBackgroundColor(this.props.statusBarColor || "#f5f7f9");
```
Comment (gcandal) [inline react/util/mixins/screen.js:17]: Why this specific value for the fallback? have you considered doing nothing if the prop is not defined?

**Review by gcandal — COMMENTED** [review]
Changelog :)

**Review by gcandal — APPROVED** [review]

**Comment by gcandal** [comment]
I'm renaming this to StatusBarMixin

**Review by joamag — APPROVED** [review]

**Pull request merged**